editor.planx.uk/src/@planx/components/PlanningConstraints/Public.test.tsx

```diff
@@ -239,4 +239,21 @@ describe("following a FindProperty component", () => {
     expect(negativeConstraintsContainer).toBeVisible();
     expect(getByRole("heading", { name: /Ecology/ })).toBeVisible();
   });
+
+  test("default disclaimer text should render if none provided", async () => {
+    const { queryByText } = setup(
+      <PlanningConstraints
+        title="Planning constraints"
+        description="Things that might affect your project"
+        fn="property.constraints.planning"
+        disclaimer=""
+        handleSubmit={vi.fn()}
+      />,
```

Comment:
nit: in August's case (and if you create a new Planning Constraints in the editor & remove the disclaimer text), we don't have `disclaimer=""` but rather the `disclaimer` prop isn't present at all ! 

That might be the more realistic case to test here?